examples/vision/learnable_resizer.py

```diff
@@ -0,0 +1,278 @@
+"""
+Title: Learning to Resize in Computer Vision
+Author: [Sayak Paul](https://twitter.com/RisingSayak)
+Date created: 2021/04/30
+Last modified: 2021/04/30
+Description: How to optimally learn representations of images for a given resolution.
+"""
+"""
+It is a common belief that if we constrain vision models to perceive things as humans do,
+their performance can be improved. For example, in [this work](https://arxiv.org/abs/1811.12231),
+Geirhos et al. showed that the vision models pre-trained on the ImageNet-1k dataset are
+biased toward texture whereas human beings mostly use the shape descriptor to develop a
+common perception. But does this belief always apply especially when it comes to improving
+the performance of vision models? 
+
+It turns out it may not always be the case. When training vision models, it is common to
+resize images to a lower dimension ((224 x 224), (299 x 299), etc.) to allow mini-batch
+learning and also to keep up the compute limitations.  We generally make use of image
+resizing methods like **bilinear interpolation** for this step and the resized images do
+not lose much of their perceptual character to the human eyes. In [Learning to Resize
+Images for Computer Vision Tasks](https://arxiv.org/abs/2103.09950v1), Talebi et al. show
```

Comment:
The cited paper is a preprint from 2 month ago with 0 citations -- do we have external guarantees of its quality and importance?

Reply:
It's hard for me to answer. The main component that intrigued me while I was implementing this one was the idea **_that maybe learning to generate better views that are better for a model to improve its performance_**. 

On the other hand, it came from Google Research. 

So, combining these two factors I put my trust.

Reply:
Ok, sounds good.